### DISCUSSION TITLE
一种进行doris数据库的本地备份的方法

### DISCUSSION THREAD

**Discussion by nospeak12345** [apache/doris#30220] [A - General / Q&A]
哈哈，标题其实是问题！
是这样的，通过broker进行备份，还需要构建HDFS之类的分布式文件系统，对于以卖萌为生的小型企业来说，会有一些复杂。
那么是否有什么方式，可以把某张doris的表进行本地备份的方式呢？这张表非常大，有上百TB，包含1000个分区，我想备份在我用NFS挂载过来的一个目录里，假设这个目录是/mnt/nfs，这个目录有200TB大小，是否有相应的方法，不用构建HDFS，S3之类的方法？
我的DORIS是PALO1.1.0.
thanks bro.

**Comment by LemonLiTree** [comment]
首先建议升级 ，备份推荐使用backup 方式
